code/modules/jobs/job_exp.dm

```diff
@@ -17,20 +18,20 @@ GLOBAL_PROTECT(exp_to_update)
 	var/isexempt = C.prefs.db_flags & DB_FLAG_EXEMPT
 	if(isexempt)
 		return 0
-	var/my_exp = C.calc_exp_type(get_exp_req_type())
-	var/job_requirement = get_exp_req_amount()
-	if(my_exp >= job_requirement)
+	var/my_exp = C.calc_exp_type(get_exp_req_type()) // this value is returned in minutes.
+	var/job_requirement = get_exp_req_amount() // this value is returned in hours.
```

Comment:
"this value is returned in hours"

it's deciseconds, right?

Reply:
*yells into pillow*

you're right